package com.java.s4.exception;

public class ExceptionMain4 {

	public static void main(String[] args) throws ChildException {
		ExStudy2 ex2 = new ExStudy2();
		
		ex2.ex1();

	}

}
